### PR TITLE
fix test tables for new clinvar assertions

### PR DESCRIPTION
**Pull request by jklugherz**
fix test cases for my accidental dev [commit](https://github.com/broadinstitute/seqr-loading-pipelines/commit/a493221cb42ff5b338b973b7768a116f6e6f9326) which adds new assertions and gold star lookup values

### CODE REVIEW

**Comment by bpblanken** [comment]
@hanars so you see this.  We encountered changes to the "assertions" modifiers that were parsing away from the pathogenicities.  If we continue to see frequent naming changes to these we should probably change the behavior to not hard-fail and instead dynamically map the enums.  We're not doing anything on the search end other than returning in the JSON response and displaying in the UI right?

**Comment by hanars** [comment]
yeah, seqr has nothing hard coded around assertions so theres no harm if these dynamically update

File: v03_pipeline/lib/tasks/reference_data/update_variant_annotations_table_with_updated_reference_dataset_test.py [hunk 1]
```diff
@@ -388,7 +388,7 @@ def test_update_vat_with_updated_rdc_mito_38(
                     clinvar_mito=None,
                     dbnsfp_mito=hl.Struct(
                         SIFT_score=None,
-                        MutationTaster_pred='N',
```
Comment (jklugherz) [inline v03_pipeline/lib/tasks/reference_data/update_variant_annotations_table_with_updated_reference_dataset_test.py:391]: fixed this column in the test table too

**Review by bpblanken — APPROVED** [review]

**Comment by bpblanken** [comment]
merging so I can pull into my other branch

**Pull request merged**